source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -0,0 +1,52 @@
+Open edX Sumac Release
```

Comment:
Does it make sense to say something about how this hasn't been released yet in the title of the page? I realize the link to get here has that, but if someone were to come to this page in some other way...?

Reply:
Yeah, that's a good call out.